docs/content/latest/explore/indexes-constraints/expression-index-ysql.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Expression Indexes
+linkTitle: Expression Indexes
+description: Using Expression Indexes in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: expression-index-ysql
+    parent: explore-indexes-constraints
+    weight: 230
+aliases:
+   - /latest/explore/ysql-language-features/indexes-1/
+   - /latest/explore/indexes-constraints/indexes-1/
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../expression-index-ysql/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+</ul>
+
+Indexes usually are created based on the columns, but the expression indexes lets you create an index based on a generic expression(function or modification of data entered) involving the table columns.
```

Comment:
Should be helpful to mention that they are also called Function Based Indexes sometimes